hazelcast/src/test/java/com/hazelcast/jet/core/SplitBrainTest.java

```diff
@@ -104,19 +101,17 @@ public void when_quorumIsLostOnMinority_then_jobDoesNotRestartOnMinorityAndCance
             long jobId = jobRef[0].getId();
 
             assertTrueEventually(() -> {
-                JetServiceBackend service = getJetServiceBackend(firstSubCluster[0]);
-                assertEquals(COMPLETED, service.getJobCoordinationService().getJobStatus(jobId).get());
+                JobCoordinationService coordinationService = getJetServiceBackend(firstSubCluster[0]).getJobCoordinationService();
+                assertEquals(COMPLETED, coordinationService.getJobStatus(jobId).get());
+                assertThat(coordinationService.isJobUserCancelled(jobId).get()).isFalse();
```

Comment:
you do not have to put `isJobUserCancelled` inside `assertTrueEventually` if you wait for it's future. BTW, it would be better if `get()` had timeout

Reply:
>you do not have to put isJobUserCancelled inside assertTrueEventually if you wait for it's future

Ah yes, I just wanted it to be close to the other assertion as those are connected.

> BTW, it would be better if get() had timeout
True in production code, but here it the whole test may just time out